scss/bootstrap/overrides/style/_modals.scss

```diff
@@ -45,3 +45,13 @@
     top: initial !important;
   }
 }
+
+.modal-backdrop {
+  z-index: 100000 !important; // Some current elements have z-index 99999
+}
+
+.modal {
```

Comment:
Wouldn't it be better at this point to have
```css
.modal,
.modal-backdrop {
  z-index: 100001 !important;
}

Reply:
They have to have different z-indexes.

Reply:
Could you give some example of which browsers can ignore DOM position?

Reply:
As far as I remember, I've seen it in FF https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context. It is still good to ensure they have different z-indexes.